.env.default

```
@@ -1,4 +1,4 @@
-DJANGO_ALLOWED_HOSTS=localhost
+DJANGO_ALLOWED_HOSTS=*
```

```suggestion
DJANGO_ALLOWED_HOSTS=localhost,.gitpod.io
```
Also suggesting to keep the restriction and be explicit about other sources.

---

I've committed the above without testing.

---

Tested, it works 👍🏻